src/qt/createassetdialog.cpp

```diff
@@ -170,7 +216,13 @@ void CreateAssetDialog::ipfsStateChanged()
 
 void CreateAssetDialog::checkAvailabilityClicked()
 {
-    QString name = ui->nameText->text();
+    QString name = "";
```

Comment:
these lines duplicate the block starting at line 160 -- could possibly be consolidated

Reply:
Consolidated into function